module/userProjects/helper.js

```diff
@@ -1952,7 +1970,18 @@ module.exports = class UserProjectsHelper {
             
             let totalCount = 0;
             let data = [];
-            
+            //this is to check if user has already joined program or not
```

Comment:
@ankitpws  why we are having this here instead in **detailsV2** ? as we discussed we need to pass rootOrganisation detail of program to front-end for calling consent api. are we doing it anywhere? please confirm

Reply:
that has been taken in solution details API.